pkg/numaresourcesscheduler/controlplane/discover_test.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controlplane
+
+import "testing"
+
+func TestDefaults(t *testing.T) {
+	info := Defaults()
+	if info.NodeCount < 1 {
```

Comment:
this is a nice bonus because the intent was to actually return a meaningful value. In general doesn't make sense to have replicas < 1 for control plane, the tests just checks that.